### ISSUE TITLE
[a11y] Custom form validation messages are not read by screenreaders

### ISSUE THREAD

**Issue by FloppyNotFound** [ngx-formly/ngx-formly#2458]
**Description**
When creating a custom form validation, the error messages, which are shown below the control, are not read out correctly via screenreaders.
I have taken a look at the original Angular Material control validation, it is working there.

It is not working with formly, because the connected aria-properties are missing.
It just reads out "required", which is not the correct validation message.

**Minimal Reproduction**
- Open https://formly.dev/examples/validation/custom-validation
- Enter an invalid email address
- Open https://material.angular.io/components/input/overview#changing-when-error-messages-are-shown
- Enter an invalid email address
- Open the Inspector (F12) and inspect both input fields and error messages

Angular Material:
![image](https://user-images.githubusercontent.com/18663555/91067046-b6550580-e632-11ea-8b6a-9bc018628987.png)

Formly:
![image](https://user-images.githubusercontent.com/18663555/91067169-da184b80-e632-11ea-9bcc-6a1de1d127ee.png)

**Your Environment**

- Angular version: 10.0.7
- Formly version: 5.9.3

**Additional context**
Tested with Formly for Angular Material

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.10.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
